### PR TITLE
Add change log and the updated pull request template

### PR DESCRIPTION
**Pull request by MohamedSabthar**
## Purpose
$title

## Remarks
ballerina-platform/ballerina-standard-library#1198

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Change Log
+This file contains all the notable changes done to the Ballerina UDP package through the releases.
+
+## [0.9.0-alpha6]  - 2021-04-02
```
Comment (BuddhiWathsala) [inline changelog.md:4]: isn't there any change that we can list out here?

Comment (MohamedSabthar) [reply changelog.md:4]: No notable changes after Alpha 3 release.

Comment (BuddhiWathsala) [reply changelog.md:4]: Then should we need this entry? @shafreenAnfar

Comment (Maninda) [reply changelog.md:4]: I think we can remove that line and add this line to the `Change Log`.

`The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).`

Comment (MohamedSabthar) [reply changelog.md]: Updated

**Review by Maninda — APPROVED** [review]

**Review by shafreenAnfar — APPROVED** [review]

**Pull request merged**